public/assets/scripts.js

```diff
@@ -0,0 +1,28 @@
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
Since the application is in React, can we just load this script from react? Do not think adding scripts to assets is a great approach.